app/graphql/types/profile.rb

```diff
@@ -3,12 +3,14 @@
 require_relative 'interfaces/rules_preload'
 require_relative 'concerns/test_results'
 require_relative 'concerns/profile_pseudo_policy'
+require_relative 'concerns/profile_scoring'
 
 module Types
   # Definition of the Profile type in GraphQL
   class Profile < Types::BaseObject
```

Comment:
There are several Profile only fields reporting scores and compliance:
* score
* compliant
* rules_passed and rules_failed
* last_scanned

I guess they should stay profile-based except the `last_scanned`. WDYT?

Reply:
Right, they are all (except `compliant`) technically on TestResult, but they can exist on profile within the context of a single host as the latest TestResult for that host and profile, which is what it means today. I noticed the `score` attribute has an _optional_ `system_id` argument :thinking: seems that should be required.